posts/inside-rust/2021-01-19-changes-to-rustdoc-team.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Changes to Rust rustdoc team"
```

Comment:
I think 'Rust' is assumed, since this is the inside-rust blog :P
```suggestion
title: "Changes to the Rustdoc team"
```

Reply:
We're never too sure!